digitalidentity/service.go

```diff
@@ -28,14 +28,16 @@ func CreateShareSession(httpClient requests.HttpClient, shareSessionRequest *Sha
 		HTTPMethod: http.MethodPost,
 		BaseURL:    apiUrl,
 		Endpoint:   endpoint,
-		Headers:    requests.AuthHeader(clientSdkId),
+		Headers:    requests.AuthHeader(clientSdkId, &key.PublicKey),
```

Comment:
it seems that we don't need the public key here; do we need the clientSDKId as we're also adding it a few rows later as a Param?

Reply:
As read from documentation backend needs this header.

Reply:
It seems that we don't need it the Params (but we need it in the Headers)